metrics/storage/webtooling.sh

```diff
@@ -51,6 +52,44 @@ Usage: $0 <count>
 EOF
 }
 
+clean() {
+	sudo systemctl restart containerd && sleep 1
+	local containers=( $(sudo ctr c list -q) )
+	for i in "${containers[@]}"; do
+		sudo ctr task kill -s SIGKILL "$i"
+		sudo ctr c rm "$i"
+	done
+}
+
+verify_task_is_completed_on_all_containers() {
+	local FILE_CMD="cat $file_path/$file_name | grep Geometric"
+	local containers=( $(sudo ctr c list -q) )
+	local completed=()
+	local count=1
+	while [ $count -le $NUM_CONTAINERS ]; do
+	    for i in "${containers[@]}"; do
+	        # Only check containers that have not completed the task at this step
+	        if [[ ! " ${completed[@]} " =~ " ${i} " ]]; then
+                    check_file_content=$(sudo ctr t exec --exec-id "$(random_name)" "$i" sh -c "$FILE_CMD")
+                    if [ ! -z "$check_file_content" ]; then
+                        ((count=count+1))
+                        completed+=("$i")
+                    fi
+                fi
+            done
+            sleep 10
```

Comment:
This is a long sleep. If it has to be this long, can you add a log call so it's clear what is happening. Also, do we need a timeout in this loop so we could log something like this each time through:

```bash
local sleep_secs=10

  :

echo "INFO: loop $count of $max: sleeping for $sleep_secs"
sleep $sleep_secs
```

Reply:
Thank you, fixed.

Reply:
@dborquez it seems you forgot to push the change?

Reply:
@Jakob-Naucke Hi, I was testing some fixes. But I'm done.

Reply:
@dborquez after making your changes, you should amend them to your commit, rebase it, and force-push it, see https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#github-workflow

Reply:
@Jakob-Naucke done, thanks.